### ISSUE TITLE
test_that("D1Client downloadObject") failing CRAN checks

### ISSUE THREAD

**Issue by jsta** [DataONEorg/rdataone#231]
```r
── 1. Error: D1Client downloadObject (@test.D1Client.R#548) ────────────────────────────────────────
trying to get slot "obsoletedBy" from an object of a basic class ("NULL") with no slots
1: downloadObject(d1cKNB, pid, path) at testthat/test.D1Client.R:548
2: downloadObject(d1cKNB, pid, path)
3: .local(x, identifier, ...)
```
See: https://cran.rstudio.com/web/checks/check_results_dataone.html

**Comment by gothub** [comment]
@jsta thanks for logging this. I've identified the cause of this error and reported it to the DataONE Cyber Infrastructure personnel, who should be able to resolve it by early next week.

**Comment by gothub** [comment]
Fixed in commit e50f4ce555a398b872be86f2f442dd61cecadf8c. Updated package is available from CRAN (v2.1.2)
